operations/_objects.md

```diff
@@ -183,3 +183,13 @@ The profile data of the user who created or last updated the record.
 | `TaxRateCode` | string | optional | Tax rate code for the item. `null` for untaxed amounts. |
 | `NetValue` | number | required | The net value that the tax is calculated from. |
 | `TaxValue` | number | required | The value of the tax. |
+
+#### Currency value
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Currency` | string | required | ISO-4217 code of the `Currency`.  |
+| `Value` | number | optional | Amount in the currency. |
+| `Net` | number | optional | The net value that the tax is calculated from. |
+| `Tax` | number | optional | The value of the tax. |
+| `TaxRate` | number | optional | Percentage of the tax rate. |
```

Comment:
I quickly searched the docs for how this type is represented and couldn't find the place where these 3 properties had been listed. It seems there is a consistent misrepresentation on how this type is documented. Maybe we should not expose it after all and replace CurrencyValueOld with CurrencyValue in the code base? TaxRate property seems dodgy, it might not be a percentage in reservations/price (just looked at the code, didn't test it) and it is a big difference in meaning.

Reply:
I agree, it may not be safe to remove them from code, but do we gain anything by bringing it to the docs (for the first time ever, it seems)? We could just redirect CurrencyValueOld to CurrencyValue in documentation generator.

Reply:
Just discussing this with @jnv, let me try and summarise:
* In answer to the question 'do we document deprecated features?' the answer from me will always be yes, if it's in the API then it should be documented
* It sounds like we have two versions of `Currency value`, one of which is effectively deprecated; so we just follow the normal process for a deprecated feature; if the two versions have different content but the same name, then I would agree with @jnv to follow the convention we established with **Get reservations** (messy as it is), so the old one can be called something like `Currency value (ver 20XX-XX-XX)`
* As to where common object definitions are located in the Docs, I leave that up to you guys because it depends on the workings of the Generator automation, I just think it should be tidy, consistent, and technically correct
* I note that at the moment `Currency value` is defined both in the **Accounting items** page and in the **Payment requests** page, and the definitions are identical, so it would be cleaner if these were put in a single location and all references made to that location
* Finally, logically it makes sense that if we have an old version and a new version of `Currency value`, these would be in the same place, whatever that is